web_translate_dialog/__manifest__.py

```diff
@@ -8,11 +8,14 @@
     "version": "13.0.1.0.0",
     "category": "Web",
     "website": "https://github.com/OCA/web",
-    "author": "Camptocamp, " "Tecnativa, " "Odoo Community Association (OCA)",
+    "author": "Camptocamp, "
+    "Tecnativa, "
+    "Odoo Community Association (OCA), "
+    "InitOS Gmbh",
```

Comment:
authors should always come before OCA. Nevertheless, you cannot add the company here since this is just a plain migration